src/SDKs/Authorization/Authorization.Tests/Graph/GraphManagementClient.cs

```diff
@@ -248,11 +248,12 @@ private HttpRequestMessage CreateRequest(string uri, HttpMethod method, string b
 
             if (this.testEnvironment != null && this.testEnvironment.TokenInfo != null)
             {
-                // Not Supported in current code
-                // var tokenCredentials = (TokenCredentials)this.testEnvironment.Credentials;
-                // httpRequest.Headers.Authorization = new AuthenticationHeaderValue(
-                //                                this.testEnvironment.AccessTokenType,
-                //                                this.testEnvironment.AuthorizationContext.AccessToken);
+				// Not Supported in current code
+				// var tokenCredentials = (TokenCredentials)this.testEnvironment.Credentials;
+				httpRequest.Headers.Authorization = new AuthenticationHeaderValue("Bearer",
+					"Put the access token for graph here for the tests to work in record mode");
+            //                                    this.testEnvironment.AccessTokenType,
+            //                                    this.testEnvironment.AuthorizationContext.AccessToken);
```

Comment:
once this pr is completed[ [https://github.com/Azure/azure-sdk-for-net/pull/3278](https://github.com/Azure/azure-sdk-for-net/pull/3278)] ,do incorporate these changes